pkg/apis/sessions/session_state.go

```diff
@@ -164,12 +164,11 @@ func into(s *string, f codecFunc) error {
 // algorithms.
 func lz4Compress(payload []byte) ([]byte, error) {
 	buf := new(bytes.Buffer)
-	zw := lz4.NewWriter(nil)
-	zw.Header = lz4.Header{
-		BlockMaxSize:     65536,
-		CompressionLevel: 0,
-	}
-	zw.Reset(buf)
+	zw := lz4.NewWriter(buf)
```

Comment:
Related to the Redis comment -- is this required by the 1.15 Go bump? Or is this better served as its own PR to keep the scope of changes separate?

Do you have a link to the v4 library for these LZ4 API changes? I'm struggling to find it.